### DISCUSSION TITLE
Reports in APIM Developer Portal

### DISCUSSION THREAD

**Discussion by restlessankyyy** [Azure/api-management-developer-portal#1539] [General]
How does reports page in APIM internally communicates with APIM?
How can I personalise reports in Developer portal?

**Comment by mikebudzynski** [comment]
What kind of customizations are you looking for? The API reports widget uses the API reports API endpoint to retrieve the data.

**Comment by restlessankyyy** [comment]
We are thinking of a dashboard solution where all the users can see the number of API calls that were successful or failures irrespective of their subscription to their Products / APIs.

**Reply by mikebudzynski** [reply]
Do you mean the total number of API calls, made by all users (not just the logged in user)?

**Reply by restlessankyyy** [reply]
Yes

**Reply by restlessankyyy** [reply]
So that they can filter it with any apis they want to see. It will promote more visibility on the APIs stability to the end customers who don't have access to HAPI Analytics board in general.

**Reply by mikebudzynski** [reply]
Can you please request this feature in the Issues section?

**Reply by restlessankyyy** [reply]
Yes of course. 👍🏼
